admins.md

```diff
@@ -0,0 +1,11 @@
+ # QuTiP Admin Team
+
+The functioning of the board of the QuTiP project can be found in the [governance document](governance.md#2-admin-team). The current members of the board are:
+
+- Shahnawaz Ahmed
+- Eric Giguere
+- Neill Lambert
+- Boxi Li
+- Jake Lishman
+- Alex Pitchford
+- Nathan Shammah
```

Comment:
Add Simon?

Reply:
This has not been voted yet, so the admin list now is correct. Can be updated later with another PR to reflect eventual updates :).